app/models/application_record.rb

```diff
@@ -19,7 +19,8 @@ class ApplicationRecord < ActiveRecord::Base
     prepend: true
 
   default_scope -> {
-    order(created_at: DEFAULT_SORT_ORDER)
+    # FIXME(ezekg) why do we need an explciit table_name everywhere?
+    order("#{table_name}.created_at": DEFAULT_SORT_ORDER)
```

Comment:
Simple answer: unions. They some unions don't have a table, since they potentially union across multiple tables (theoretically — but we always union the same table, iirc). This — arguably ugly — explicitness makes sure that we don't hit ambiguous column errors with a bare `created_at` ordering.